.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,42 @@
+name: CodeQL Analysis
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 8 * * *'
+  workflow_dispatch:
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: windows-latest
+    steps:
+      - name: Setup .NET SDK
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 2.1.505
+
+      - name: Add MSBuild to PATH
```

Comment:
Does `dotnet` CLI not work even under `windows-latest` ?

Reply:
Yes, we established that together

Reply:
@szlatkow alright, add an explanation about that to the PR. It is odd that the `dotnet` CLI doesn't work with the project on Windows and the owners should know about it.